tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -31,3 +31,21 @@ Check If POD Exists
     ${status}     ${val}       Run keyword and Ignore Error   OpenShiftCLI.Get   kind=Pod     namespace=${namespace}   label_selector=${label_selector}
     [Return]   ${status}
 
+Verify Operator Pod Status
+    [Documentation]    Verify Pod status
+    [Arguments]  ${namespace}   ${label_selector}  ${expected_status}=Running
+    ${status}    Get POD Status    ${namespace}    ${label_selector}
+    Run Keyword IF   $status != $expected_status     Fail     RHODS operator status is not matching with the exepected state
```

Comment:
typo in the error message, should be `expected state`

Reply:
Done